sources/packages/backend/apps/api/src/route-controllers/application-offering-change-request/_tests_/e2e/application-offering-change-request.aest.controller.assessApplicationOfferingChangeRequest.e2e-spec.ts

```diff
@@ -98,6 +106,90 @@ describe("ApplicationOfferingChangeRequestAESTController(e2e)-assessApplicationO
     ).toBe(applicationOfferingChangeRequest.requestedOffering.id);
   });
 
+  it("Should access the student appeal of the current assessment for the provided application offering change request id.", async () => {
+    // Arrange
+    const application = await saveFakeApplication(db.dataSource, undefined, {
```

Comment:
Please take a look inside `saveFakeApplication`, the method already creates `User`, `Student`, and `Offering`.

Reply:
Is there a reason why this method is still using `createFakeStudentAssessment` and `saveFakeApplicationOfferingRequestChange`? Are those two records not created by the `saveFakeApplication`?

Reply:
I'll remove `createFakeStudentAssessment` since it has been created by `saveFakeApplication`, but `saveFakeApplicationOfferingRequestChange` creates a record for ApplicationOfferingChangeRequest, which is not created by `saveFakeApplication`.

Reply:
Sorry, I meant `createFakeEducationProgramOffering` instead of `saveFakeApplicationOfferingRequestChange`.